client/core/exchangeratefetcher.go

```diff
@@ -196,7 +199,7 @@ func fetchMessariRates(ctx context.Context, log dex.Logger, assets map[uint32]*S
 		slug := strings.ToLower(asset.Symbol)
 		reqStr := fmt.Sprintf(messariURL, slug)
 
-		if err := getInto(ctx, reqStr, res); err != nil {
+		if err := getRates(ctx, reqStr, res); err != nil {
 			log.Error(err)
```

Comment:
We'll have to see how this goes.  I think it's gonna be irritating (to me) to see the expected 429 code all the time, but perhaps with 12 minute intervals it won't be a big deal.